package.json

```diff
@@ -1,12 +1,7 @@
 {
   "private": true,
-  "workspaces": [
-    "clients/tabby-agent",
-    "clients/vscode",
-    "clients/vim",
-    "clients/intellij"
-  ],
   "engines": {
     "node": ">=18"
-  }
+  },
+  "packageManager": "^pnpm@8.3.1"
```

Comment:
It should be `"packageManager": "pnpm@8.3.1"` ?
The prefix `^` seems to be causing `Usage Error: Unsupported package manager specification (^pnpm@8.3.1,^pnpm,8.3.1)`

Reply:
Certainly, actually I think this PR is still in progress, and  if we plan to migrate to pnpm we also need to  update the CI jobs scripts and documentation together, I'll continue working on it if you agree this enhancement